test/python/compiler/test_assembler.py

```diff
@@ -735,53 +735,70 @@ def test_assemble_backend_rep_times_delays(self):
         """Check that rep_time and rep_delay are properly set from backend values."""
         # use first entry from allowed backend values
         rep_times = [2.0, 3.0, 4.0]  # sec
-        rep_delays = [2.5e-3, 3.5e-3, 4.5e-3]
+        rep_delay_range = [2.5e-3, 4.5e-3]
+        default_rep_delay = 3.0e-3
         self.backend_config.rep_times = rep_times
-        self.backend_config.rep_delays = rep_delays
```

Comment:
There aren't any backends anymore which would support `rep_delays`, is that right? They all moved to `rep_delay_range`?

Reply:
yes.